Makefile

```diff
@@ -23,11 +23,27 @@ clean:
 	rm -f node_modules/.uptodate .pydeps
 	rm -rf build
 
+.PHONY: env
+env:
```

Comment:
I kinda feel like a comment summarizing what is happening here would be useful for those not ninjas in stream editing.